pkgs/by-name/wo/world-wall-clock/package.nix

```diff
@@ -0,0 +1,38 @@
+{
+  lib,
+  fetchFromGitHub,
+  python3,
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "world-wall-clock";
+  version = "0.1.4";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "ddelabru";
+    repo = pname;
```

Comment:
```suggestion
    repo = "world-wall-clock";
```
#277994

Reply:
Made the repo name explicit instead of referencing `pname`